### ISSUE TITLE
[SIN] Pre-process the iris dataset and implement different Machine Learning algorithms

### ISSUE THREAD

**Issue by AjayKhalsa** [CMPN-CODECELL/Hacktoberfest2020#8]
- Preprocess and perform EDA on the iris dataset.
- Implement ML algorithms like
- A few example solutions:
  - Logistic Regression

  - Random Forest Classifier

  - SVM Classifier

  - Decision Tree

  - K-Means

  - Gradient Boosting (XGBoost, CatBoost)

  - Neural networks

  - Other

**Comment by Trimankaur** [comment]
@AjayKhalsa I would like to work on this issue.

**Comment by AjayKhalsa** [comment]
@Trimankaur Go ahead feel free to make  PR when you're ready
